site/css/chart.scss

```diff
@@ -19,7 +19,7 @@
 
     // Charts shouldn't be below this height in any circumstance, whether mobile or not.
     // Except in iframes, where we override this.
-    min-height: 480px;
+    min-height: 540px;
```

Comment:
The minimum height of Grapher is increased a little since many charts did not really fit into a 480px-tall space.